modules/Store.js

```diff
@@ -0,0 +1,32 @@
+export default class Store {
+  static getLists = () => {
+    let Lists;
+    if (localStorage.getItem('Lists') === null) {
+      Lists = [];
+    } else {
+      Lists = JSON.parse(localStorage.getItem('Lists'));
+    }
+    return Lists;
+  }
+
+  static setLists = (list) => {
+    const Lists = this.getLists();
+    Lists.push(list);
+    localStorage.setItem('Lists', JSON.stringify(Lists));
+  }
+
+  static deleteLists = (deleted) => {
+    const Lists = this.getLists();
+    const filteredLists = Lists.filter(
+      (listItem) => listItem.index !== deleted,
+    );
+
+    // update index of list
+
+    filteredLists.forEach((filteredList, index) => {
+      filteredList.index = index + 1;
+    });
+
+    localStorage.setItem('Lists', JSON.stringify(filteredLists));
+  }
+}
```

Comment:
- [ ] [Optional] Remember that one of the major requirements for this milestone is to implement the feature of editing a single task and saving it to the `localStorage`. It would be nice to do this so that your application functions properly. 👍